docs/netavark-firewalld-forwarding.5.md

```diff
@@ -0,0 +1,85 @@
+% NETAVARK-FIREWALLD 5 Netavark Firewalld Interactions Man Page
```

Comment:
The name here is different from the actual file name `netavark-firewalld-forwarding` NETAVARK-FIREWALLD seems good enough so I suggest you rename the file.

Second, section 5 is wrong. That is for file formats/config files per [man-pages(7)](https://man7.org/linux/man-pages/man7/man-pages.7.html). IT should be in section 7 based on that.